src/AdoPat/IStorageWrapper.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.Authentication.AdoPat
+{
+    /// <summary>
+    /// An interface for a thin, testable wrapper around MSAL's Storage class.
+    /// </summary>
+    public interface IStorageWrapper
+    {
+        /// <summary>
+        /// Read and unprotect cache data.
+        /// </summary>
+        /// <returns>Unprotected cache data.</returns>

```

Comment:
Curious why we call these data unprotected and protected

Reply:
Just copying what the underlying library refers to them as. The data is usually encrypted by the MSAL cache helpers, so I think that's why this language is used.